lib/src/view/design_main_strand.dart

```diff
@@ -552,7 +552,12 @@ after:
     items[num_bases_idx] = DialogInteger(
         label: 'number of bases', value: 5, tooltip: 'number of bases to include in this extension');
 
-    var dialog = Dialog(title: 'add extension', items: items, type: DialogType.add_extension);
+    var dialog = Dialog(title: 'add extension', items: items, type: DialogType.add_extension, process_saved_response: (saved_items) {
```

Comment:
Actually, can we re-do this to keep the original behavior? The original behavior was actually to save nothing from the previous dialog. There's also logic that only gives an option to add an extension where there isn't currently one. For example, if there's a 5' extension and no 3' extension, the original behavior was for there only to be one option for where to add the extension, 3'.

For this it might be simplest to add an optional keyword argument to the Dialog constructor that simply tells it not to use any saved responses when first rendering the dialog, i.e., just go with what the original Dialog object says.